### DISCUSSION TITLE
How do I troubleshoot any variant of hotkey shortcuts not working across multiple devices (Windows/Linux) on firefox

### DISCUSSION THREAD

**Discussion by fiskhest** [darkreader/darkreader#6835] [Q&A] [answered]
As the title suggests, the hotkeys used to disable the extension for the current site *or* globally all of a sudden does not work for me (they did before). To be sure that I do not have conflicting keybinds, I tried changing the assigned hotkey shortcuts to no result. 

How can this be troubleshooted? I tried disabling a couple of addons that could be culprits to no effect.

**Comment by Gusted** [comment, marked as answer]
https://github.com/darkreader/darkreader/issues/6832

**Reply by Gusted** [reply]
Note: Its a regression. Hotfix will be deployed later this week.

**Reply by fiskhest** [reply]
Thanks!
